server/controllers/companyController.js

```diff
@@ -32,8 +34,13 @@ const createCompany = async (req, res) => {
 
     // add doc to db
     const { name, industry, location, size, email, password } = req.body;
+
+        //handeling errors 
+        //still empty
+
     try {
-        const company = await Company.create({ name, industry, location, size, email, password })
+        const company_id=req.company._id
```

Comment:
```suggestion
        const { company_id } = req
```

Also:
does this line need to be in a try catch block?